app/views/recipes/show.html.erb

```diff
@@ -1,12 +1,12 @@
-<h3>Public?</h3>
+
```

Comment:
- Please add the information on whether the recipe is public or not. It's in the wireframe. You can copy from the public recipes page.